hi.py

```diff
@@ -0,0 +1,8 @@
+def hello():
+  return "Hello User" 
+
+    
+    
+    
+if name == "__name___"
+    hello()
```

Comment:
[![Panto AI](https://raw.githubusercontent.com/pantomaxdotco/assets/refs/heads/main/pr-comment-branding-50.png)](https://www.pantomax.co)
Indentation error. The line should be properly indented under the if condition.